sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsServiceSerializer.cs

```diff
@@ -196,26 +196,29 @@ private static string ReadModelVersion(JsonElement documentElement)
 
         private static TextDocumentBatchStatistics ReadDocumentBatchStatistics(JsonElement documentElement)
         {
-            if (documentElement.TryGetProperty("statistics", out JsonElement statisticsElement))
+            if (!documentElement.TryGetProperty("statistics", out JsonElement statisticsElement))
             {
-                int documentCount = default;
-                int validDocumentCount = default;
-                int invalidDocumentCount = default;
-                long transactionCount = default;
+                return default;
+            }
 
-                if (statisticsElement.TryGetProperty("documentsCount", out JsonElement documentCountValue))
-                    documentCount = documentCountValue.GetInt32();
-                if (statisticsElement.TryGetProperty("validDocumentsCount", out JsonElement validDocumentCountValue))
-                    validDocumentCount = validDocumentCountValue.GetInt32();
-                if (statisticsElement.TryGetProperty("erroneousDocumentsCount", out JsonElement erroneousDocumentCountValue))
-                    invalidDocumentCount = erroneousDocumentCountValue.GetInt32();
-                if (statisticsElement.TryGetProperty("transactionsCount", out JsonElement transactionCountValue))
-                    transactionCount = transactionCountValue.GetInt64();
+            var documentDict = new Dictionary<string, long>()
```

Comment:
By putting the values in a Dictionary, we are allocating this in the heap, while the way we had it before, each string was in the stack.
Performance wise, it will be preferable to leave the code with the 4 if statements.

Reply:
i'm with you. 

thank you for you review!